submissions/adeolaFafemi/php/loopsAndNumbers.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+for($i=100; $i<150; $i++){
+     if($i>100){
```

Comment:
I think there is a way to avoid this check. You don't have to do this `if` statement